OpenRA.Mods.Common/Traits/Render/Hovers.cs

```diff
@@ -13,6 +13,7 @@
 using System.Collections.Generic;
 using System.Drawing;
 using System.Linq;
+using OpenRA;
```

Comment:
This doesn't seem to be necessary for compiling.

Reply:
yeah, seems I previously used it for something, can't recall what